app/src/main/java/in/testpress/testpress/models/Exam.java

```diff
@@ -224,9 +224,13 @@ public String formatDate(String inputString) {
         SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
         simpleDateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
         try {
-            date = simpleDateFormat.parse(inputString);
-            DateFormat dateformat = DateFormat.getDateInstance();
-            return dateformat.format(date);
+            if(inputString != null && !inputString.isEmpty()) {
```

Comment:
This fix is not associated with the issue. Please raise a separate issue and put it in a separate commit.